app/controllers/concerns/sufia/collections_controller_behavior.rb

```diff
@@ -30,6 +30,16 @@ def show
       presenter
     end
 
+    def create
+      super
+      ActiveFedora::Base.find(params[:batch_document_ids]).each(&:update_index) unless params[:batch_document_ids].nil?
```

Comment:
I'm concerned about performance implications here. Loading a bunch of objects from Fedora, one at a time, and then calling `#update_index` on each seems like it will hang up creates and updates. Perhaps this should be done in a background job. @jcoyne what do you think?